src/index.js

```diff
@@ -55,10 +55,11 @@ function App() {
         <Route exact path="/login" component={LoginPage} />
         <Route path="/implicit/callback" component={LoginCallback} />
         {/* any of the routes you need secured should be registered as SecureRoutes */}
+
         <SecureRoute
           path="/"
           exact
-          component={() => <MyProfile LoadingOutlined={LoadingOutlined} />}
+          component={() => <DashboardPage LoadingOutlined={LoadingOutlined} />}
```

Comment:
Nice! Good thinking to get the user to land on the Dashboard after login as opposed to their Profile page.

Reply:
Like Monica said Good idea changing the routing.